doc_admin/AdminManual.adoc

```diff
@@ -135,24 +115,38 @@ NOTE: The information needed to use the OH software is not included in this manu
 
 == 2 Installation
 
+Open Hospital is a client application that can be used in two modalities: 
```

Comment:
`used in two modalities:` might be a little clearer as `used in two different ways:`   This makes it just a little more readable.

Reply:
good, thanks!